pkgs/development/guile-modules/guile-ssh/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, guile, libssh
+, autoreconfHook, libtool, pkgconfig, texinfo, which
+}:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "guile-ssh";
+  version = "0.12.0";
+
+  src = fetchurl {
+    url = "https://github.com/artyom-poptsov/${pname}/archive/v${version}.tar.gz";
+    sha256 = "1775zgyc1rx9n6lfwbwi0yqsf72w84db29px9n17l62r0bx11dnm";
+  };
+
+  nativeBuildInputs = [ autoreconfHook libtool pkgconfig texinfo which ];
```

Comment:
```suggestion
  nativeBuildInputs = [ autoreconfHook pkg-config texinfo which ];
```

Reply:
Thanks, I appreciate those were all fairly obvious. Updated and rebased into the individual commits